instrumentation/cassandra/cassandra-4.0/library/src/main/java/io/opentelemetry/instrumentation/cassandra/v4_0/CassandraTelemetryBuilder.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.cassandra.v4_0;
+
+import io.opentelemetry.api.OpenTelemetry;
+
+public class CassandraTelemetryBuilder {
+
+  static final String DEFAULT_INSTRUMENTATION_NAME = "io.opentelemetry.cassandra";
+
+  private final OpenTelemetry openTelemetry;
+
+  private String instrumentationName = DEFAULT_INSTRUMENTATION_NAME;
+  private boolean statementSanitizationEnabled;
+
+  public CassandraTelemetryBuilder(OpenTelemetry openTelemetry) {
+    this.openTelemetry = openTelemetry;
+  }
+
+  public CassandraTelemetryBuilder setInstrumentationName(String instrumentationName) {
```

Comment:
Sorry I'm not understanding here. I saw that this `KafkaInstrumenterFactory` is like a builder and it is receiving the instrumentationName with the constructor. Where should it be used? 

- Not in the `CassandraTelemetryBuilder` otherwise we should pass the instrumentation_name name as parameter (then what is the sense of having the Factory class?).
- Not in the singleton (where `KafkaInstrumenterFactory` is used) instead of the builder, otherwise what is the sense of having the builder?

Reply:
I left unchanged the `cassandra-4.0` and in `cassandra-4.4` the instrumentation name cannot be set (it's defaulted).